README.md

```diff
@@ -37,7 +37,15 @@ generally recommended for CI usage in particular.
 To install, download the binary for your platform from "Assets" and place this
 into your `$PATH`:
 
-On Mac & Linux:
+On Mac:
+
+```console
+curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-darwin-amd64
```

Comment:
I think that $(uname) returns `darwin` on mac, no need to differentiate

Reply:
it returns `Darwin`, however, https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-Darwin-amd64 works just fine, and we unconditionally save it as `./kind`

Reply:
I got 404